### PR TITLE
Add visual indicator when the game is muted.

### LINKED ISSUE #17549: Add visual indicator when sound is muted
The hotkey to mute sounds could be pressed accidentally and in the heat of battle the player might not notice the chat notification and wonder why the sound is off. Displaying a persistent visual indicator would be helpful to quickly assert whether the sound has been muted.

See  https://github.com/OpenRA/OpenRA/pull/17482#discussion_r362676920

### PR DESCRIPTION
**Pull request by pchote**
Fixes #17549.

### CODE REVIEW

**Review by atlimit8 — DISMISSED** [review]
The mute indicator disappears with the rest of the battlefield control UI when a mission is finished, but that is not important. It does come back with the rest of the battlefield control UI when stay is selected. Otherwise, it looks fine to me.

File: OpenRA.Mods.Common/Widgets/Logic/MuteIndicatorLogic.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/MuteIndicatorLogic.cs]: None of the above are actually used.

**Comment by pchote** [comment]
Rebased (first push) and fixed (second).

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d080d5d2f2e473c742f8043d34dc072d5f01463c...c32a62c437e2a5336451aa117c2bfb665a259e51)

**Pull request merged**